gr-blocks/lib/correctiq_swapiq_impl.h

```diff
@@ -0,0 +1,41 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+#ifndef INCLUDED_CORRECTIQ_SWAPIQ_IMPL_H
+#define INCLUDED_CORRECTIQ_SWAPIQ_IMPL_H
+
+#include <gnuradio/blocks/correctiq_swapiq.h>
+
+#define SWAPTYPE_FLOATCOMPLEX 1
```

Comment:
not a strong opinion, but an `enum` might be more robust here than these macros.